lib/reek/ast/object_refs.rb

```diff
@@ -1,33 +1,30 @@
 module Reek
-  module AST
+  module AST  # :nodoc:
+    ObjectRef = Struct.new(:name)
```

Comment:
Since everything under AST is also private, you can move the `# @api private` on line 7 up to above line 2.